web_duplicate_visibility/views/assets.xml

```diff
@@ -14,7 +14,7 @@
     <template id="qunit_suite"
               name="web_duplicate_visibility qunit assets"
               inherit_id="web.qunit_suite">
-        <xpath expr="//html/head"
+        <xpath expr="//script[last()]"
```

Comment:
The code is right. Did you check on runbot?

Reply:
@JayVora-SerpentCS  first thanks for review :),
yes i did it was working well